src/IcuEC/IcuConvAutoConfigDialog.cs

```diff
@@ -19,24 +19,47 @@ public class IcuConvAutoConfigDialog : SilEncConverters40.AutoConfigDialog
 		[DllImport("IcuConvEC.dll", EntryPoint = "IcuConvEC_ConverterNameList_start", CallingConvention = CallingConvention.Cdecl)]
 		static extern unsafe int CppConverterNameList_start();
 
-#if __MonoCS__
-		[DllImport("IcuConvEC.dll", EntryPoint = "IcuConvEC_ConverterNameList_next", CallingConvention = CallingConvention.Cdecl)]
+        static string CppConverterNameList_next()

```

Comment:
Wrong indentation - but I guess Jason already mentioned that.

Reply:
Done